common/src/main/java/de/florianmichael/rclasses/common/memory/Memory.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * This file is part of RClasses - https://github.com/FlorianMichael/RClasses
+ * Copyright (C) 2023 FlorianMichael/EnZaXD and contributors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.florianmichael.rclasses.common.memory;
+
+import java.lang.reflect.Field;
+
+/**
+ * This class represents functions to work with native memory
+ */
+public class Memory {
+    /**
+     * The unsafe instance.
+     */
+    public final static sun.misc.Unsafe UNSAFE = getUnsafe();
+
+    /**
+     * @return the unsafe instance
+     */
+    public static sun.misc.Unsafe getUnsafe() {
```

Comment:
The IOUtils already contains an unsafe instance, creating a second util for it in another sub-module doesn't look like a good design choice.

Reply:
I don't see any connection between an IO module and `sun.misc.Unsafe`. `sun.misc.Unsafe` is an API for native memory management and has nothing to do with IO.

Reply:
If you would've looked into the IO module you would know it's containing unsafe utils and native access stuff / IO utils, Common really doesn't seem like a good place for it no matter if IO is better or not, and if there is already a usage of Unsafe, you could have moved it completely or followed my suggestion, but duplicating this util doesn't seem like a good solution at all.